kubemarine/resources/schemas/definitions/services.json

```diff
@@ -54,6 +54,9 @@
     "etc_hosts": {
       "$ref": "services/etc_hosts.json"
     },
+    "etc_hosts_generated": {
```

Comment:
It can be redefined but it makes no sense because `services.etc_hosts_generated` is set empty just before filling with nodes information. It can be used for a quick reference in cluster_finalized.yaml if necessary, though.
Final `etc_hosts` and `coredns_configmap.yaml` are dumped into ./dump/ folder and can be also revised there.